src/components/verticaltabs/_verticaltabs.scss

```diff
@@ -0,0 +1,55 @@
+.cmp-verticaltabs{
+    font-size: $font-m;
+    margin: $field-margin;
+    position: relative;
+  
+  &__tabs-container{
+    display: flex;
+    margin-top: 16px;
+  }
+  &__tabpanel{
+    flex: 1;
+    padding: 16px 56px;
+  }
+  &__tabpanel--active{
+    
+  }
+  &__tablist{
```

Comment:
Create mixin for tablist and tab

Reply:
Do we need this mixin it is used here only and will not be used for other components?